app/controllers/review_mapping_controller.rb

```diff
@@ -513,6 +513,18 @@ def response_report
       end
     end
 
+  def save_grade_and_comment_for_reviewer
```

Comment:
Assignment Branch Condition size for save_grade_and_comment_for_reviewer is too high. [18.68/15]